packages/lodestar/src/api/rest/options.ts

```diff
@@ -0,0 +1,60 @@
+import {ApiNamespace} from "../index";
+import {booleanOption, IConfigurationModule} from "../../util/config";
+import {processApiNamespaces} from "../utils";
+
+export interface IRestApiOptions {
+  enabled: boolean;
+  api: ApiNamespace[];
+  host: string;
+  cors: string;
+  port: number;
+}
+
+export default {
+  enabled: false,
```

Comment:
Rest is part of the spec, should we enable it by default?

Reply:
It says in spec to allow node manager to decide. Generally its safest to leave it disabled because some users might not wanna serve validators.